test/functional/tests/incremental_load/test_incremental_load.py

```diff
@@ -28,20 +35,134 @@
+                TestRun.fail(f"Core {core.core_id} should be detached but is {core.get_status()}.")
+
+    with TestRun.step("Load cache"):
+        cache = casadm.load_cache(cache_dev)
+        if cache.get_status() is not CacheStatus.running:
+            TestRun.fail(f"Cache {cache.cache_id} should be running but is {cache.get_status()}.")
+
+        for core in core_pool_list:
+            if core.get_status() is not CoreStatus.active:
+                TestRun.fail(f"Core {core.core_id} should be active but is {core.get_status()}.")
+
+
+@pytest.mark.CI
+@pytest.mark.os_dependent
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_incremental_load_missing_core_device():
+    """
+    title: Incremental load test with missing core device
+    description: |
+        Test incremental load and core pool functionality with missing core partition
+    pass_criteria:
+      - cores are in detached state after adding to core pool
+      - visible cores should be in active state after start and load
+      - core with missing device should be in detached state after load
+    """
+
+    with TestRun.step("Partition cache and core devices"):
+        cache_device = TestRun.disks["cache"]
+        core_device = TestRun.disks["core"]
+
+        cache_device.create_partitions([Size(2, Unit.GibiByte)])
+        core_device.create_partitions([Size(4, Unit.GibiByte)] * 3)
+
+        cache_dev = cache_device.partitions[0]
+        core_dev_list = core_device.partitions
+
+    with TestRun.step("Start cache and add cores"):
+        cache = casadm.start_cache(cache_dev=cache_dev, force=True)
+        core_list = [cache.add_core(core_dev=core_dev) for core_dev in core_dev_list]
+
+    with TestRun.step("Check if all cores have active status"):
+        for core in core_list:
+            if core.get_status() is not CoreStatus.active:
+                TestRun.fail(f"Core {core.core_id} should be active but is {core.get_status()}.")
+
+        missing_core = cache.get_core_devices()[-1]
+
+    with TestRun.step("Stop cache."):
+        cache.stop()
+
+    with TestRun.step("Remove last core partition"):
+        core_device.remove_partition(core_device.partitions[-1])
+
+    with TestRun.step("Load cache"):
+        cache = casadm.load_cache(cache_dev)
+        if cache.get_status() is not CacheStatus.incomplete:
+            TestRun.fail(
+                f"Cache {cache.cache_id} should be incomplete but is " f"{cache.get_status()}."
+            )
+        for core in cache.get_core_devices():
+            if core.get_status() is not CoreStatus.active:
+                TestRun.fail(f"Core {core.core_id} should be Active but is {core.get_status()}.")
+        if missing_core.get_status() is not CoreStatus.inactive:
+            TestRun.fail(f"Remove core should be Inactive but is {missing_core.get_status()}.")
```

Comment:
```suggestion
            TestRun.fail(f"Core without backend should be Inactive but is {missing_core.get_status()}.")
```
or `Core with missing device`